src/main/java/org/folio/innreach/batch/contribution/service/ContributionJobRunner.java

```diff
@@ -114,12 +115,13 @@ public void runInitialContribution(InstanceIterationEvent event, String topic) {
       log.info("Initial: cannot contribute record for null event, contribution id: {}", contributionId);
       return;
     }
-    log.info("Initial: processing instance iteration event: {}", event);
+    log.info("Initial: processing instance iteration event instance id: {}", event.getInstanceId());
 
     var instanceId = event.getInstanceId();
-
-    if (isUnknownEvent(event, iterationJobId)) {
-      log.info("Initial: skipping unknown event, current job is {}", iterationJobId);
+    boolean isUnknownEvent = isUnknownEvent(event, iterationJobId);
```

Comment:
As we are having a logger inside the if condition , do we really need to store the value in a variable and log that value ? Moreover , the below line will print like this **Initial: isUnknownEvent: true** or **Initial: isUnknownEvent: false** which will not be useful for us I feel.